lib/ansible/module_utils/network/skydive/api.py

```diff
@@ -74,6 +82,114 @@ def __init__(self, **kwargs):
                 env = ('SKYDIVE_%s' % key).upper()
                 if env in os.environ:
                     kwargs[key] = os.environ.get(env)
+
+
+class skydive_inject_protocol(object):
```

Comment:
The simplest test that could be done (more or less the example in the documentation) fails. skydive_inject_protocol' object has no attribute 'connection_made'". You have decided to not extend WsClientDefaultProtocol but WSClient needs to implement the protocol interface for trollius. As expected the above exception is raised.